incubator/cassandra/templates/statefulset.yaml

```diff
@@ -181,7 +181,6 @@ spec:
       name: data
       labels:
         app: {{ template "cassandra.name" . }}
-        chart: {{ template "cassandra.chart" . }}
         release: {{ .Release.Name }}
         heritage: {{ .Release.Service }}
```

Comment:
One last thing: I think you can delete this one as well: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#persistentvolumeclaim

Keeping it won't cause issue, but it may be better to delete.

Reply:
Which one? `heritage`? Or the entire `labels`?

Reply:
heritage